### PR TITLE
add character encoding meta

### PR DESCRIPTION
**Pull request by mehdirahimi**
The character encoding of the page must be declared.

### CODE REVIEW

**Review by montogeek — APPROVED** [review]

File: src/content/configuration/mode.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ var config = {
   //...
 };
 
-module.exports = (env, argv) => {
+module.exports = (env, argv) => ({
```
Comment (chenxsan) [inline src/content/configuration/mode.md:167]: The original version has no problem, only when we're returning an object should we wrap the `{}` with `()`.

Comment (mehdirahimi) [reply src/content/configuration/mode.md:167]: It's a SyntaxError

**Comment by chenxsan** [comment]
@mehdirahimi Hi, can you please revert the second commit so I can merge your pull request? As I mentioned here https://github.com/webpack/webpack.js.org/pull/4070#discussion_r506782781, there's no bug in the original version. If you're not convinced, check this demo https://codesandbox.io/s/async-pine-p9h41?file=/src/index.js of your version, you shall see `SyntaxError`.

**Review by chenxsan — APPROVED** [review]

**Comment by mehdirahimi** [comment]
> @mehdirahimi Hi, can you please revert the second commit so I can merge your pull request? As I mentioned here [#4070 (comment)](https://github.com/webpack/webpack.js.org/pull/4070#discussion_r506782781), there's no bug in the original version. If you're not convinced, check this demo https://codesandbox.io/s/async-pine-p9h41?file=/src/index.js of your version, you shall see `SyntaxError`.

It's done

**Pull request merged**